addons/binding/org.openhab.binding.onebusaway/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: OneBusAway Binding
+Bundle-SymbolicName: org.openhab.binding.onebusaway;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
```

Comment:
Yes, you are right, see https://github.com/eclipse/smarthome/pull/3278 if you like a review is more than welcome!